main.go

```diff
@@ -1,9 +1,14 @@
 package main
 
 import (
+	"fmt"
+
 	"github.com/dachad/tcpgoon/cmd"
 )
 
+var version string
```

Comment:
you should also expose this by a -version flag, right?

Reply:
I've exposed this as a tcpgoon command, but it could be another flag...
however, I would keep it simple and avoid an extra flag

Reply:
a command is good! If you request the version, it does not make sense to use other flags related to the execution...